apps/vmq_gojek_auth/src/vmq_gojek_auth.erl

```diff
@@ -43,13 +43,16 @@
                  vmq_gojek_auth_acl_read_all,
                  vmq_gojek_auth_acl_write_all,
                  vmq_gojek_auth_acl_read_user,
-                 vmq_gojek_auth_acl_write_user
+                 vmq_gojek_auth_acl_write_user,
```

Comment:
Let's not use `vmq_gojek_auth` as the name for this plugin, something more generic should work.

Reply:
`vmq_auth`?

Reply:
This module is already present inside vmq_server. You cannot have duplicate named modules. Can we go with vmq_enhanced_auth?

Reply:
I think it can work, but if all features of vmq_auth are in vmq_enhanced_auth, we might as well just keep vmq_enhanced_auth and call it to vmq_auth. I'll leave the decision up to you now.